packages/flutter/test/foundation/error_reporting_test.dart

```diff
@@ -102,16 +102,16 @@ Future<void> main() async {
       exception: getAssertionErrorWithLongMessage(),
     ));
     expect(console.join('\n'), matches(
-      '^══╡ EXCEPTION CAUGHT BY FLUTTER FRAMEWORK ╞═════════════════════════════════════════════════════════\n'
-      'The following assertion was thrown:\n'
-      'word word word word word word word word word word word word word word word word word word word word\n'
-      'word word word word word word word word word word word word word word word word word word word word\n'
-      'word word word word word word word word word word word word word word word word word word word word\n'
-      'word word word word word word word word word word word word word word word word word word word word\n'
-      'word word word word word word word word word word word word word word word word word word word word\n'
-      '\'[^\']+flutter/test/foundation/error_reporting_test\\.dart\':\n'
-      'Failed assertion: line [0-9]+ pos [0-9]+: \'false\'\n'
-      '════════════════════════════════════════════════════════════════════════════════════════════════════\$',
+      r'^══╡ EXCEPTION CAUGHT BY FLUTTER FRAMEWORK ╞═════════════════════════════════════════════════════════\n'
```

Comment:
ditto - the `\n` at the end here is important.

Reply:
not really in a regexp. See my above comment.